main/main.c

```diff
@@ -0,0 +1,78 @@
+#include "discovery_beacon.h"
+#include "driver_setup.h"
+#include "esp_log.h"
+#include "http_server.h"
+#include "persistent_settings.h"
+#include "socketcand_server.h"
+
+// Name that will be used for logging
+static const char *TAG = "main";
+
+void app_main(void) {
+  // Create an event loop.
+  ESP_ERROR_CHECK(esp_event_loop_create_default());
+
+  // Initialize the networking stack.
+  ESP_ERROR_CHECK(esp_netif_init());
+
+  // Wait for hardware to start
+  vTaskDelay(pdMS_TO_TICKS(200));
+
+  // Initialize nonvolatile memory storage
+  ESP_ERROR_CHECK(persistent_settings_init());
+
+  // Set up the long-press settings reset button
+  ESP_ERROR_CHECK(persistent_settings_setup_reset_button());
+
+  // Read saved settings
+  persistent_settings_t settings;
+  ESP_ERROR_CHECK(persistent_settings_load(&settings));
+
+  // Print the settings over uart
+  char settings_json[512];
```

Comment:
Another variable that would be good to statically allocate

Reply:
Fixed in 72219c3!